### PR TITLE
fix(native): Fixed link url in native app

### PR DESCRIPTION
**Pull request by piotrski**
This should fix elk-zone/elk-native/issues/37

### CODE REVIEW

File: components/status/StatusActionsMore.vue [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <script setup lang="ts">
+import { env } from 'process'
+import { metaProperty } from '@babel/types'
```
Comment (patak-dev) [inline components/status/StatusActionsMore.vue:3]: Are these imports needed?

Comment (piotrski) [reply components/status/StatusActionsMore.vue]: No, it was just ts trying to be too clever. I've already removed it.

**Review by danielroe — CHANGES_REQUESTED** [review]
I can see we might have situations like this throughout the app. Maybe a feature we can toggle on in the tauri module that forces the `elk.zone` hostname so we don't have to reference tauri here?

**Comment by piotrski** [comment]
@danielroe Because it's just a browser behavior, and Tauri has no control over it. I guess the only thing we could do is to monkey patch `location.origin` in the native bundle so that it always returns `elk.zone` but I'm not a fan of that solution.

**Comment by danielroe** [comment]
I understand the need and agree. I'm suggesting that rather than check the environment variable here, we instead have a global setting (maybe in app config) that changes behaviour across _everywhere_ in the app that accesses `location.origin`.

**Pull request closed without merging**